src/components/LMMedia/LMLinkPreview/types.ts

```diff
@@ -0,0 +1,29 @@
+import {ImageStyle, TextStyle, ViewStyle} from 'react-native';
+
+// data model for ogTags object in attachmentMeta
+interface OgTagsProps {
+  description: string;
+  title: string;
+  url: string;
+  image?: string;
+}
+
+// data model for attachmentMeta
+interface AttachmentMeta {
+  entityId: string;
+  ogTags: OgTagsProps;
+}
```

Comment:
move this interface to different folder, and then use it here and use builder pattern for creating this class.